src/ZODB/DB.py

```diff
@@ -401,12 +401,6 @@ def __init__(self, storage,
           - `xrefs` - Boolian flag indicating whether implicit cross-database
             references are allowed
         """
-        if isinstance(storage, six.string_types):
-            from ZODB import FileStorage
-            storage = ZODB.FileStorage.FileStorage(storage, **storage_args)
-        elif storage is None:
-            from ZODB import MappingStorage
-            storage = ZODB.MappingStorage.MappingStorage(**storage_args)
```

Comment:
Why the removal of the convenience args? I know they don't seem to be documented but I bet they're in use. They don't seem to hurt much.